### ISSUE TITLE
Always getting the first 6 Episodes in Search

### ISSUE THREAD

**Issue by JonnyKreng** [PCJones/MediathekArr#11]
When trying to find Episodes >7 of "Das Traumschiff"  only get the first 6 Episodes in the Interactive Search.  Automatic search isn't working either.

Logs:
```
info: Program[0]
      Incoming Request: GET /api?t=tvsearch&extended=1&cat=5040,5030,5000&tvdbid=133371&season=1&ep=7&limit=100&offset=0
info: Program[0]
      Incoming Request: GET /api?t=tvsearch&extended=1&cat=5040,5030,5000&season=1&ep=7&q=Das+Traumschiff&limit=100&offset=0
info: Program[0]
      Incoming Request: GET /api?t=tvsearch&extended=1&cat=5040,5030,5000&season=1&ep=7&q=Love+Boat+DE&limit=100&offset=0
info: Program[0]
      Incoming Request: GET /download/api?mode=queue&start=0&limit=0&category=tv&apikey=x&output=json
info: Program[0]
      Incoming Request: GET /download/api?mode=history&start=0&limit=60&category=tv&apikey=x&output=json
```

Interactive Search:
![image](https://github.com/user-attachments/assets/a35a05de-99a5-45c9-b32d-b4f1b82f6903)
![image](https://github.com/user-attachments/assets/3c125432-7608-4695-a7ef-cf87a2a52b20)

**Comment by PCJones** [comment]
I'll let you know once it's fixed, thanks for reporting! If you find any other shows that aren't working please let me know

**Comment by PCJones** [comment]
Fixed with 1.0: https://github.com/PCJones/MediathekArr/releases/tag/v1.0-alpha.1

![grafik](https://github.com/user-attachments/assets/2ae3eac4-24aa-48e4-b708-4feed35665e6)
